src/Ztm.WebApi.Tests/SqlCallbackRepositoryTests.cs

```diff
@@ -0,0 +1,210 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using Xunit;
+using Ztm.Data.Entity.Contexts.Main;
+using Ztm.Data.Entity.Testing;
+
+namespace Ztm.WebApi.Tests
+{
+    sealed class TestCallbackResult : CallbackResult
```

Comment:
We should move this class after `SqlCallbackRepositoryTests` for readability due to the main class of the file should come first.